### ISSUE TITLE
Can't process certificates that ends with new line

### ISSUE THREAD

**Issue by jiri-lazensky** [joe-elliott/cert-exporter#155]
Cert-exporter can't process certificates that ends with new line.

Error example:
```
I0103 13:48:29.196561       1 periodicSecretChecker.go:164] Publishing <cert> tls.crt
E0103 13:48:29.196578       1 periodicSecretChecker.go:167] Error exporting secret Failed to parse intermediate as a per
```

I believe that this line is the "cause".

https://github.com/joe-elliott/cert-exporter/blob/28bb9e554f835f8940f89e2390b9bc205c76f100/src/exporters/certHelpers.go#L101

In my forked repo I have solved it by adding function that will remove all new lines from the end of file - https://github.com/joe-elliott/cert-exporter/compare/master...jiri-lazensky:cert-exporter:feat-remove-newline-from-end-of-the-cert.

Please let me know if you consider this as issue, or you expect that certificates will end without new line char.

**Comment by jiri-lazensky** [comment]
@joe-elliott I guess that would work also 👍. 
I went with simplest sanitation I could think of without touching the cert parsing itself.

**Comment by joe-elliott** [comment]
yeah, if you're up for a PR i think just ignoring whitespace would be better that way we'd catch similar future issues

**Comment by jiri-lazensky** [comment]
#156

**Comment by joe-elliott** [comment]
Nope. I actually thought this was in a release. I will get one cut soon to catch the fixes and improvements that have been added since 

https://github.com/joe-elliott/cert-exporter/releases/tag/v2.12.0
